server/data/educationAndWorkPlanClient.ts

```diff
@@ -7,6 +7,13 @@ export default class EducationAndWorkPlanClient {
     return new RestClient('Education and Work Plan API Client', config.apis.educationAndWorkPlanApi, token)
   }
 
+  async createInduction(prisonNumber: string, createRequest: CreateInductionRequest, token: string): Promise<unknown> {
```

Comment:
I'm conscious that the existing `ciagApiClient` returns a `Promise<CiagPlan>`, but our API endpoint doesn't do this, so we need to call the GET Induction endpoint separately - TBD...

Reply:
I suspect it'll be fine as you have it. Yes, the current `ciagApiClient` returns a `Promise<CiagPlan>`, but I suspect there isnt any code that actually reads/uses that value, so I suspect we'll be fine as we are 👍